client/components/ui/data-table/components/data-table-header.tsx

```
@@ -56,38 +53,46 @@ export function DataTableHeader<TData>({
-                                                    header.column.getIsResizing() && "bg-blue-600 w-[2px]"
-                                                }`}
-                                            />
-                                        </div>
-                                    )}
-                            </TableHead>
-                        ))
+                                                onMouseDown={header.getResizeHandler()}
+                                                onTouchStart={header.getResizeHandler()}
+                                                className="absolute top-0 right-0 h-full w-3 cursor-col-resize select-none group/resizer z-10"
+                                                onClick={(e) => e.stopPropagation()}
+                                            >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)